Dockerfile

```diff
@@ -0,0 +1,13 @@
+# syntax=docker/dockerfile:1
+
+FROM node:18-alpine
+
+WORKDIR /app
+
+COPY . .
+
+RUN yarn install
+
+EXPOSE 3000
+
+CMD ["yarn", "dev"]
```

Comment:
For production there should be optimized docker image. First build an app using build command and then pass the build to the server. In most cases it is nginx because it is small. Then it is pushed to server